Assets/ProBuilder Data/ProBuilderPreferences.asset

```diff
@@ -12,22 +12,14 @@ MonoBehaviour:
   m_Name: ProBuilderPreferences
   m_EditorClassIdentifier: 
   m_Bool_keys: []
-  m_Int_keys:
```

Comment:
In the future try not to commit changes like these that are irrelevant to the feature you're adding.